drivers/ethernet/eth_native_posix.c

```diff
@@ -208,7 +207,7 @@ static int eth_send(struct net_if *iface, struct net_pkt *pkt)
 
 	update_gptp(iface, pkt, true);
 
-	SYS_LOG_DBG("Send pkt %p len %d", pkt, count);
+	NET_DBG("Send pkt %p len %d", pkt, count);
```

Comment:
why is this required? Why not sue LOG_DBG directly?

Reply:
I had some issues when using logger directly. I will revisit this in next version.